roles/ceph-mon/tasks/start_monitor.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: activate monitor with upstart
+  file: path=/var/lib/ceph/mon/ceph-{{ ansible_hostname }}/{{ item }}
+        state=touch
+        owner=root
+        group=root
+        mode=0600
+  with_items:
+    - done
+    - upstart
+  when: ansible_distribution == "Ubuntu"
+  changed_when: false
+
+- name: start and add that the monitor service to the init sequence (ubuntu)
+  service: name=ceph-mon
+           state=started
+           enabled=yes
+           args="id={{ ansible_hostname }}"
```

Comment:
If y'all are open to reformatting, the preferred style for new code landing in Ursula is like this:
```
- name: ensure ceph-mon service enabled and started
  service:
    name: ceph-mon
    enabled: yes
    args: id={{ ansible_hostname }}
```

This is a pure YAML style and it solves some problems with template quoting, with complex arguments (passing lists), etc. 

Not every task in Ursula is this way right now, obviously, but as we add new tasks or modify existing tasks this is the preferred format.